kafka/adminapi.go

```diff
@@ -555,6 +705,14 @@ type DescribeACLsResult struct {
 // DeleteACLsResult provides delete ACLs result or error information.
 type DeleteACLsResult = DescribeACLsResult
 
+// GroupTopicPartitions represents a consumer group's TopicPartitions.
```

Comment:
Can we move GroupTopicPartitions to kafka.go, and then create a new struct here and embed GroupTopicPartitions in this PR itself?
Since, if later we need to add something, the API for ListConsumerGroupOffsets etc will change and that will be undesirable.

Reply:
Yes, call it ListConsumerGroupOffsetsResult or AlterConsumerGroupOffsetsResult as in J. 
and embed GroupTopicPartitions so we can extend that if needed